### PR TITLE
docs changes for npm registry fix

### PR DESCRIPTION
**Pull request by mhoeger**

### CODE REVIEW

File: docs/KNOWN_ISSUES.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Known Issues
```
Comment (annaji-msft) [inline docs/KNOWN_ISSUES.md:1]: Should we add this to FAQs.md?

File: docs/QUICKSTART.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ You can explore all currently available connector SDK's [on this repo itself](ht
 For more information on the connector Actions that are available, see the [Connectors reference documentation](https://docs.microsoft.com/connectors/connector-reference/). You can also explore [in depth how-to guides for some connectors like Bing Search](https://docs.microsoft.com/en-us/azure/connectors/connectors-create-api-bingsearch
 ).
 
-### Set up Azure GitHub package registry
+### Set up authentication to Azure GitHub package registry
```
Comment (annaji-msft) [inline docs/QUICKSTART.md:26]: nit: Authenticate by logging in to npm using the npm login command. When prompted, enter your GitHub username for Username, your personal access token for Password, and your public email address for Email: 

in this comment can you mention "personal access token(PAT)"

File: docs/QUICKSTART.md [hunk 3]
```diff
@@ -81,6 +78,8 @@ Install the connector you want to use. For example:
 </details>
 
 ## Create a connection
+> IMPORTANT: Connection creation is currently supported in West Central US only. Fix ETA: 10/5/20
```
Comment (annaji-msft) [inline docs/QUICKSTART.md:81]: bold west central us

**Review by annaji-msft — APPROVED** [review]

**Comment by mhoeger** [comment]
Closing in favor of: https://github.com/Azure/Connectors/pull/55

**Pull request closed without merging**